site/content/docs/main/node-agent-concurrency.md

```diff
@@ -32,7 +32,7 @@ At least one node is expected to have a label with the specified ```RuledConfigs
 If one node falls into more than one rules, e.g., if node1 also has the label ```beta.kubernetes.io/instance-type=Standard_B4ms```, the smallest number (3) will be used.  
 
 ### Sample
-A sample of the complete ```node-agent-config``` configMap is as below:
+A sample of the complete configMap is as below:
```

Comment:
Complete configmap should contain
```
kind: ConfigMap
metadata:
...
```
IMO. Something user can copy/paste to a file and create.

Reply:
The sample is a json instead of a yaml, and and we are using `kubectl create cm` to create it, so the sample could be used by directly copy/paste.

Reply:
Thanks for the clarification.